.gitignore

```diff
@@ -114,4 +114,6 @@ gradle-app.setting
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
-replay_pid*
\ No newline at end of file
+replay_pid*
+
+*.ser
```

Comment:
are you commiting this for a reason, if not i don't think it should be a part of the pr

Reply:
Yes, to prevent people committing instance specific data files to the git repo.